### PR TITLE
code clean up - removing old comments

### PR DESCRIPTION
**Pull request by MAZamarripa**
## Fixes/Addresses/Summary/Motivation:

(Replace this with the issue # fixed or addressed in this PR as well as any other related PRs.  If
no issue or PRs exist, then add a brief statement on what changes this PR is bringing in.)

## Changes proposed in this PR:
- updated imports, updated Enum method, clean code - removed old comments
-

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my
contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md file
   at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has
   rights to intellectual property that includes these contributions, I represent that I have
   received permission to make contributions and grant the required license on behalf of that
   employer.

### CODE REVIEW

**Review by tarnold17 — DISMISSED** [review]
Looks good to me. My only question is what is the purpose of adding the three html files to the tests directory?

File: pareto/operational_water_management/operational_produced_water_optimization_model.py [hunk 1]
```diff
@@ -3601,26 +3300,26 @@ def BeneficialReuseDeliveriesRule(model, o, t):
 
 
 def water_quality(model, df_sets, df_parameters):
-    # Add parameter for water quality at each pad
+    # TODO: Add parameter for water quality at each pad
```
Comment (melody-shellman) [inline pareto/operational_water_management/operational_produced_water_optimization_model.py:3303]: these are not TODOs. They describe each line of code. I will update to clarify!

File: pareto/strategic_water_management/strategic_produced_water_optimization.py [hunk 2]
```diff
@@ -1636,8 +1450,7 @@ def create_model(df_sets, df_parameters, default={}):
         units=model.decision_period,
         doc="Pipeline construction/expansion lead time [time]",
     )
-
-    # COMMENT: It is expected that the units for this parameter are hours, which usually differs from the time units used
+    # TODO: It is expected that the units for this parameter are hours, which usually differs from the time units used
```
Comment (melody-shellman) [inline pareto/strategic_water_management/strategic_produced_water_optimization.py:1453]: This also is not a TODO, I will update to be clearer

**Review by melody-shellman — COMMENTED** [review]
Love this PR! A few things:
-some of the new marked "TODO's" are not actually TODOs. I will update them here shortly to add clarity (I wrote most of these comments that were mistaken for TODOs, haha)
-I have the same questions as Travis. Is there a reason for adding the "first bar.html", "first_sankey.html", "first_scatter.html"? We haven't added in the past as these are outputs from test procedures.

**Review by tarnold17 — APPROVED** [review]

**Review by melody-shellman — APPROVED** [review]
LGTM

**Pull request merged**